src/notifications/types.ts

```diff
@@ -17,3 +19,14 @@ export type NotificationConfig = {
   modes: NotificationConfigMode[];
   groups: NotificationConfigGroup[];
 };
+
+export enum PushNotificationPayloadType {
+  fareContractExpiry = 'FARE_CONTRACT_EXPIRY',
+}
```

Comment:
@KristianSelnas @rosvik 
I think it might be more scalable if the push notification payloads was phrased based on where they should be navigated in the app, and not on the cause of the navigation? Then we can add new push notifications which may reuse previously registered payloads without feeling "hackish".